packages/@glimmerx/storybook/.babelrc.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  presets: [
+    ['@babel/preset-env', { shippedProposals: true, useBuiltIns: 'usage', corejs: '3' }],
+    '@babel/preset-typescript',
+  ],
+  plugins: [
+    [
+      '@babel/plugin-proposal-decorators',
+      {
+        legacy: true,
+      },
+    ],
+    ['@babel/plugin-proposal-class-properties', { loose: true }],
+    '@babel/plugin-proposal-export-default-from',
+    '@babel/plugin-syntax-dynamic-import',
+    ['@babel/plugin-proposal-object-rest-spread', { loose: true, useBuiltIns: true }],
+    'babel-plugin-macros',
+    ['emotion', { sourceMap: true, autoLabel: true }],
```

Comment:
Are all of these needed for glimmerx or some of them are for storybook?

Reply:
I cleaned up the unused packages!